packages/contracts-bedrock/contracts/L2/L1Block.sol

```diff
@@ -74,5 +96,8 @@ contract L1Block is Semver {
         basefee = _basefee;
         hash = _hash;
         sequenceNumber = _sequenceNumber;
+        batcherHash = _batcherHash;
```

Comment:
Why did this become `batcherHash` instead of just `address _batcher`?

Reply:
Because we are doing the versioned batcher hash as per the design doc, we wanted it to be forward compatible with different batcher identification.